android/samples/sample-multi-view/src/main/java/com/google/android/filament/multiview/MainActivity.kt

```diff
@@ -406,10 +406,18 @@ class MainActivity : Activity() {
                 // If beginFrame() returns false you should skip the frame
                 // This means you are sending frames too quickly to the GPU
                 if (renderer.beginFrame(swapChain!!, frameTimeNanos)) {
+                    scene.skybox = skybox0
                     renderer.render(view0)
+
+                    scene.skybox = skybox1
```

Comment:
As an alternative to maintaining multiple skybox objects, can multi-view apps simply have 1 skybox that they mutate between calls to  `renderer.render(view)`?  In this particular demo, the background color of the skybox could be adjusted before each call to render().

If this is allowed, then I suggest that we add a default Skybox to Scene.  This would greatly reduce the amount of boilerplate that apps need to write.

Reply:
yes. will do.